### DISCUSSION TITLE
建议在页面变化时重新加载脚本的部分功能

### DISCUSSION THREAD

**Discussion by MasterOfStar** [the1812/Bilibili-Evolved#2692] [功能建议 / Ideas]
Chromium Edge 96.0.1054.53 
脚本 2.0.7
_新版播放界面未测试_
在旧版播放界面**仅使用此脚本**的情况下，合集换P（测试用的是凉风阅片无数第二季的合集）并不会维持宽屏，而是会变成常规大小的播放器。
在普通视频页面，在默认宽屏的情况下，关闭宽屏，并单击打开任意推荐视频，视频在当前页面打开时并不会变成宽屏播放（即脚本功能出现了和合集页面一样的问题）
建议在该功能上增加一个while循环，监听视频标题文本或者网址URL的变化，在发生变化后重新变为宽屏播放

（URL方案的话像AntiBV、bv2av之类的脚本也会用History.replaceState()的API改动URL，所以可能标题方案更合适？）

**Comment by the1812** [comment]
这个需要做个选项的, 有些人只需要一开始的时候宽屏, 有些人需要换 P 也宽屏

**Reply by MasterOfStar** [reply]
所以这只是个建议😂
